pom.xml

```diff
@@ -168,15 +142,9 @@
       <activation>
         <activeByDefault>true</activeByDefault>
       </activation>
-      <properties>
-        <osgi.scope>compile</osgi.scope>
-      </properties>
```

Comment:
I don't think we need the profiles any longer

Reply:
that's why it's red 😅